contracts/JBTiered721Delegate.sol

```diff
@@ -647,10 +647,29 @@ contract JBTiered721Delegate is JB721Delegate, Ownable, IJBTiered721Delegate, IE
 
       // Increment the leftover amount.
       unchecked {
-        creditsOf[_data.beneficiary] = _leftoverAmount + _stashedCredits;
+        // Keep a reference to the amount of new credits.
+        uint256 _newCredits = _leftoverAmount + _stashedCredits;
+
+        // Emit the change in credits.
+        // when _leftoverAmount != 0 there won't be a scenario where _credits < _newCredits
```

Comment:
this comment seems off (payer==beneficiary, minting until having less credit left)